server/pkg/api/server_add_instance.go

```diff
@@ -79,9 +79,18 @@ func (s *ShoesLXDMultiServer) AddInstance(ctx context.Context, req *pb.AddInstan
 	}
 	op, err := client.UpdateInstanceState(instanceName, reqState, "")
 	if err != nil {
+		// Do rollback
```

Comment:
When using the LXD API, I would like to output logs before and after to detect if it is hanging.

Reply:
fixed https://github.com/whywaita/shoes-lxd-multi/pull/30/commits/28e263693af5a04fc0838afff2898c8e8b063441